src/GoodsList.tsx

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
+import cn from 'classnames';
 import { Good } from './types/Good';
 
 type Props = {
   goods: Good[]
 };
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
-  <ul>
-    {goods.map(good => (
-      <li key={good.id} data-cy="good">
-        {good.name}
-      </li>
-    ))}
-  </ul>
+export const GoodsList: React.FC<Props> = React.memo(
+  ({ goods }) => (
+    <ul>
+      {goods.map(good => (
+        <li
+          key={good.id}
+          data-cy="good"
+          className={cn({
+            'has-text-primary': good.color === 'green',
+            'has-text-link': good.color === 'blue',
+            'has-text-danger': good.color === 'red',
+          })}
+        >
```

Comment:
you can use the style tag, maybe it will be a bit more logical because our product array has a color property. But its up to you)
```suggestion
					styles ={{color : `${good.color}`}}
        >
```

Reply:
Thank you! You are right